operator/main.go

```diff
@@ -124,6 +119,17 @@ func main() {
 	setupManager(flagVar, controllers.NewCacheFunc(), scheme)
 }
 
+func configLogger() logr.Logger {
```

Comment:
based on this issue: https://github.com/uber-go/zap/issues/485#issuecomment-359662417